New responsive website

cetully12 opened the pull request:
First PR in designing new responsive website

allella committed:
Delete Old Jekyll and Related Files (#19)

* Remove old, unused Jekyll files and directories

* Remove old, unused SASS files and directories

allella committed:
Copy Dave Matthews work to the new-website branch and remove some additional old javascript files from the old site (#20)

allella committed:
Remove unused screen.css. Move CSS to header (#21)

cetully12 committed:
Reorganized folders, started header design

cetully12 committed:
Adjusted header, img format

cetully12 committed:
Adjusted nav ul

cetully12 committed:
Added table icons, status button

cetully12 committed:
Created status symbol categories, fixed footer

cetully12 committed:
Created join-slack icon, adjusted logo

JamilQureshi11 commented:
I need work website designing

cetully12 committed:
Started dropdown menu, project blocks

allella commented:
Thanks. Here are initial notes.

Moving the apple-icon and favicon files is okay IF the associated <link> tags for each file includes the directory in the <head>. So, you'll want to update the following to include the /img directory prefix.
![image](https://user-images.githubusercontent.com/1777776/220984650-6ef07c6a-73ff-4fc3-afbf-8786e9e631f0.png)


When favicons started it was just 1 file for the web browsers and it was convention to leave that file in the root directory and the various browsers would look for favicon.ico or favicon.png even if there was no <link> tag explicitly defining where the file existed.

Then, the iPad started expecting larger icon files for it's own purposes, like the icons used if someone adds a shortcut on their iPad's main screen going to a website.

The ms-icon and android-icon files are likely used for similar Microsoft and Android specific purposes. Since those don't have <link> definitions for their custom /img location then those icons may not work.  This isn't really important, but you could research if/how those ms-icon and android-icon files are used and figure out if it's worth defining <link> tags for any of them.  The reason all of these files exist is that we used a generator tool to build all the possible favicon / icons and I'm not entirely sure if they all are still supported by their respective MS / Apple / Android apps.

Alternatively, we could just leave all those files in the root / directory and if any browsers or apps want them, then they will find them purely due to them existing where they look by default.

allella commented:
@cetully12 the "Projects" table only shows one project. I see there is HTML commented out, so it is probably a matter of removing those comment markers and modifying the reset of the table HTML for the other projects.
![image](https://user-images.githubusercontent.com/1777776/220986573-d9e839be-fd03-4470-bdd5-82fa72155377.png)

allella commented:
References to us being a "brigade of Code For America" can be removed.  While it's not official, Code For America is on a course to make all Code For CITYNAMEHERE chapters independent, as I mentioned in our Slack #random channel on Feb 9th.

allella commented:
When I click the "hamburger" menu in the top-right corner it opens the menu on the opposite side of the page, which I can only assume is a bug.

![image](https://user-images.githubusercontent.com/1777776/220987140-08c224c2-7289-455a-a199-338f7a7ed182.png)

allella commented:
@cetully12 you can just keep committing and pushing changes to your existing cetully12/new-website and this PR will update automatically to include those new changes / commits.

allella commented:
Also, I wonder if the "hamburger" menu is really useful for a single page site. It only links to outside resources and we may be best either keeping a header with the same links, or else putting them immediately below the large "hero" photo so they are front and center.

cetully12 commented:
@allella, thanks for reviewing! I updated the icon pathway, changed the style of the project list (for mobile view as of now), and removed references to being a Code for America brigade.  

I was fiddling with the hamburger menu yesterday and ran out of steam on figuring out how to correctly position the menu 🤦🏼‍♀️ I'm not attached to it for this page and like the idea you mentioned of an "external links or partners" section.

I'll keep plugging away on the tablet and desktop screen sizes too.

cetully12 committed:
Reformated project list

JamilQureshi11 commented:
![Uploading IMG_20230224_144848.jpg …]()

JamilQureshi11 commented:
![IMG_20230224_144839.jpg](https://user-images.githubusercontent.com/126048452/221192752-8c5ed366-b34e-4118-8b7a-9acd2bcb0b03.jpg)

cetully12 committed:
Created functional button menu for mobile view

cetully12 committed:
Adjusted hero-image size

cetully12 committed:
Fixed tablet screen banner formatting

cetully12 committed:
Created project blocks for tablet view

cetully12 committed:
Reformatted banner section

cetully12 committed:
Added nav bar menu to tablet/desktop screens

cetully12 committed:
Updated desktop media query

allella commented:
@cetully12 this is looking good.

I'll take a harder look, but is it ready to merge as far as you're concerned?

cetully12 committed:
Added hover to nav li, project icons, footer icons

cetully12 commented:
@allella, just about ready to merge! 

Once you review and approve, I can delete the original project table code. I commented it out to keep around just in case it was preferred to the current design.

cetully12 committed:
Removed table

allella commented:
@cetully12 if you're ready we can merge this PR and deal with the CFG renaming in a new PR.

cetully12 committed:
Changed title

cetully12 committed:
Updated heading

cetully12 commented:
Sounds good! Let’s merge 👍🏻

On Fri, Mar 17, 2023 at 3:35 PM Jim Ciallella ***@***.***>
wrote:

> @cetully12 <https://github.com/cetully12> if you're ready we can merge
> this PR and deal with the CFG renaming in a new PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/codeforgreenville/codeforgreenville.github.io/pull/23#issuecomment-1474315426>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWIOY2ZLMIAJ3MFHSHI3LZDW4S4HFANCNFSM6AAAAAAVDKK4GY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>
-- 
Colleen

allella commented:
This is live. Thanks for the work on it.

Once we confirm if we're going to use OpenUpstate we can do another PR for the name changes.  

Also, I noticed this is missing a meta description tag, so I'll add an issue to add one.
